odoo/tools/misc.py

```diff
@@ -1080,19 +1081,35 @@ def popmap(self):
 class OrderedSet(MutableSet):
     """ A set collection that remembers the elements first insertion order. """
     __slots__ = ['_map']
+
     def __init__(self, elems=()):
-        self._map = OrderedDict((elem, None) for elem in elems)
+        self._map = dict.fromkeys(elems)
```

Comment:
You cannot modify it, because in Python 3.6, the order of a dict is not guarantee in the Python documentation.